picosystem/src/display.rs

```diff
@@ -128,18 +139,19 @@ impl Display {
         self.start_flush();
     }
 
-    pub fn enable_backlight(&mut self) {
-        self.backlight_pin.set_high().unwrap();
+    pub fn enable_backlight(&mut self, delay_source: &mut impl DelayUs<u32>) {
+        self.st7789.set_backlight(st7789::BacklightState::On, delay_source).unwrap();
     }
-
-    pub fn disable_backlight(&mut self) {
-        self.backlight_pin.set_low().unwrap();
+    
+    pub fn disable_backlight(&mut self, delay_source: &mut impl DelayUs<u32>) {
+        self.st7789.set_backlight(st7789::BacklightState::Off, delay_source).unwrap();
     }
 
     pub fn wait_for_vsync(&mut self) {
-        if self.last_vsync_time != 0 && time::time_us() - self.last_vsync_time > 16_000 {
```

Comment:
Did these logs cause issues?

Reply:
It was cluttering the output with vsync missed messages. However, the FPS counter shows 59 FPS most of the time, except when loading new tiles in mathemagic.